os.md

```diff
@@ -0,0 +1,15 @@
+# Operating Systems available for Graviton based instances
+
+ Name | Version | [LSE Support](https://github.com/aws/aws-graviton-getting-started/blob/master/optimizing.md#locksynchronization-intensive-workload) | Kernel page size | AMI | Metal support | Comment
+------ | ------ | ----- | ----- | ----- | ----- | -----
+Amazon Linux 2 | 2.26-35 | Yes | 4KB | Yes | 
```

Comment:
AMI link missing, yes is wrong column

Reply:
Fixed column + added a page to explain how to use AWS Systems Manager Parameter store to get the latest AMI_IDs.